playbooks/cron.yml

```diff
@@ -0,0 +1,8 @@
+---
+- hosts: common
```

Comment:
`common` is not defined in the Ansible inventory anywhere (the `hosts` file), so I think this won't work -- did you test it?

I think you can probably target this better by doing the following, keeping it all within a playbook:

```yaml
---
- hosts: baremetal
  tasks:
    - name: Add common cron jobs
      include_role:
        name: common
        defaults_from: cron
        tasks_from: cron
    - name: Add fluentd restart job
      cron:
        minute: 0
        hour: "*/6"
        job: "docker restart fluentd | /usr/bin/logger -t {{ job_name }} && {{ cron_notification }} {{ job_name }}"
      vars:
        job_name: restart-fluentd
```

That's probably the cleanest place to keep it for now. It's a bit awkward b/c you have to know that the cron.yml playbook affects something like fluentd. Better for this to really live in kolla-ansible, but let's get something out to start.

Reply:
I believe I added a common to site-configs on dev.

Reply:
This can be done, but the ES curator doesn't play nicely with this structure because of the template.